### ISSUE TITLE
Deprecated plugin install script

### ISSUE THREAD

**Issue by champain** [CyVerse-Ansible/ansible-logstash#2]
In `plugins.yml` plugins are installed to logstash with the following binary path 

```
{{logstash_base_dir}}/bin/plugin
```
The contents of that file in the default version installed are simply a pointer to `logstash-plugin` and the devs indicate this will be deprecated in subsequent releases. Here is the content of the `plugin` binary in logstash 2.3

```
#!/bin/sh

echo "The use of bin/plugin is deprecated and will be removed in a feature release. Please use bin/logstash-plugin."

unset CDPATH
sh "$(cd `dirname $0`/..; pwd)/bin/logstash-plugin" "$@"
```

The time for this deprecation is here with logstash 5.0. Even if you don't plan on using logstash 5.0, moving this binary command will work with 2.3. I'll create a PR for this. It should be a simple fix. I think renaming the plugin binary in the plugins task should be sufficient, though users of logstash 5.0 and later may need to make sure they are using the path `/usr/share/logstash` as the `logstash_base_dir`

**Comment by strootman** [comment]
I mentioned this in #3, but I am planning on supporting `v5.0` and above for this role.

There are upcoming changes that will be extremely difficult to maintain backwards compatibility with `2.x`

**Comment by champain** [comment]
That would be fine by me. We are currently running 5+, but the small change I've attempted to introduce with that PR in #3 doesn't specifically break 2.3 at least. And the "parameterization" of `logstash_base_dir` makes it possible to retain backwards compatibility for this version of the role at least.

I am very interested in what other changes you'll be making for logstash 5.0+, though.

**Comment by strootman** [comment]
Resolved with #3
